### DISCUSSION TITLE
How to change the plot line style of multiple plot lines?

### DISCUSSION THREAD

**Discussion by ManusGit9** [spectrochempy/spectrochempy#480] [Q&A]
How can I change the plot line style for individual lines, plotted by the plot_multiple( method="scatter", ... ) method?

My aim is to increase readability of two lines, that are lying close to each other.
I tried changing the the lines_linestyle variable, but it seems to have no effect at all on my plots.

I saw the cycler, used with the colours in the PlotPreferences.axes_prop_cycle variable and was looking for something similar for line styles.

Thank you in advance.

**Comment by fernandezc** [comment]
Hello
Thanks for starting this discussion. I will try to look into this problem as soon as possible. But I can already say that this part of Spectrochempy has not been worked on much for the moment. If you have a notion of matplotlib, it is sometimes faster to develop your own scripts, but your question is an opportunity for me to come back to this part.

**Comment by atravert** [comment]
Hello,

I am working on a minor modification of the `scp.plot()`. This method currently uses a colormap which can make the distinction between two successive lines difficult.
With this modification, one would have to set "cmap=None" , to get a plot with the default cycling of colors of matplotlib:
```
import spectrochempy as scp
D = scp.read("irdata/nh4y-activation.spg")
D.plot(cmap=None)
```
this would yield this plot:
![image](https://user-images.githubusercontent.com/10098314/193265282-a23bc098-145f-4601-9ccf-14f8a3552056.png)

instead of the default:
![image](https://user-images.githubusercontent.com/10098314/193265820-0bfd3e74-3502-443d-94c5-6f134dc8ea56.png)

@ManusGit9 : does this such a modification solve your issue ?

Best regards,
Arnaud
